pkg/kn/commands/source/list_test.go

```diff
@@ -188,8 +188,9 @@ func newSourceUnstructuredObj(name, apiVersion, kind string) *unstructured.Unstr
 			"spec": map[string]interface{}{
 				"sink": map[string]interface{}{
 					"ref": map[string]interface{}{
-						"kind": "Service",
-						"name": "foo",
```

Comment:
Only kind and name are not enough here. An error will appear: "Unable to get the Subscriber's URI"